src/openrct2-ui/UiContext.cpp

```diff
@@ -963,6 +942,32 @@ class UiContext final : public IUiContext
             return;
         }
     }
+
+    InputEvent GetInputEventFromSDLEvent(SDL_Event& e)
```

Comment:
Would it make sense to do `const SDL_Event&` here?

Reply:
Yes, it should be `const`.

Reply:
```suggestion
    InputEvent GetInputEventFromSDLEvent(const SDL_Event& e)
```